### DISCUSSION TITLE
How to set velocity of wind in FDS simulation of car fire?

### DISCUSSION THREAD

**Discussion by HaodongSun-cumt** [firemodels/fds#13062] [Q&A]
Hello everyone!
I want to simulate a car fire on a bridge. The wind needs to be considered. There are some ways to set wind velocity, which one should I choose in this situation?
1)&WIND SPEED = 1,  DIRECTION = 180/
2)&SURF ID='WIND', VEL =-1, PROFILE='ATMOSPHERIC'/
3)&SURF ID='WIND', VEL =-1/
4)2)&SURF ID='WIND', VEL =-1, PROFILE='PARABOLIC'/

**Comment by rmcdermo** [comment]
Probably 1) is sufficient.  If you have detailed measurements about the wind fields, then you can incorporate them using a RAMP or PROFILE.

**Reply by HaodongSun-cumt** [reply]
Thanks so much rmcdermo. I tried 1) and 2), but the result was totally different. 
Here is case1.fds and case2.fds. The wind in case1 was set by &WIND, in case2 was set by &SURF.
[case1.txt](https://github.com/user-attachments/files/15856260/case1.txt)
[case2.txt](https://github.com/user-attachments/files/15856261/case2.txt)
The fire was different, the angle of inclination in case1 was higher.
![image](https://github.com/firemodels/fds/assets/62577923/cc626ac7-79f9-464f-916d-c9fb9171184a)
![image](https://github.com/firemodels/fds/assets/62577923/01575b15-92c5-487c-a6c5-8587507f7558)
